magento/magento2
# MC-23553: Invalid Template Strings when Translate Inline enabled

### **Bug P2**


### Description (*)
This is a usability issue.


### Fixed Issues (if relevant)
1. Fixes magento/magento2/issues/32000

### Manual testing scenarios (*)

Preconditions:

1. Magento installed
2. Several products and a customer is created
3. Add product to cart on storefront
4. Login to admin area
5. Go to stores > Configuration > Advanced > Developer > Translate Inline
6. Set Enabled for Storefront option to Yes and save configuration
7. Flush Cache 
8. Template strings are shown with formatting when inline translation is enabled.

Expected: Text remains the same
Actual: Text shows formatting characters like {{ }}

### Contribution checklist (*)
